src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
```

Comment:
[NAMING] - The name `copyState` could be improved to `stateCopy` to better represent that it is a copy of the state object.